Mathlib/LinearAlgebra/BilinearForm/TensorProduct.lean

```diff
@@ -44,13 +49,8 @@ variable (R A) in
 Note this is heterobasic; the bilinear form on the left can take values in an (commutative) algebra
 over the ring in which the right bilinear form is valued. -/
 def tensorDistrib : BilinForm A M₁ ⊗[R] BilinForm R M₂ →ₗ[A] BilinForm A (M₁ ⊗[R] M₂) :=
-  ((TensorProduct.AlgebraTensorModule.tensorTensorTensorComm R A M₁ M₂ M₁ M₂).dualMap
-    ≪≫ₗ (TensorProduct.lift.equiv A (M₁ ⊗[R] M₂) (M₁ ⊗[R] M₂) A).symm
-    ≪≫ₗ LinearMap.toBilin).toLinearMap
-  ∘ₗ TensorProduct.AlgebraTensorModule.dualDistrib R _ _ _
-  ∘ₗ (TensorProduct.AlgebraTensorModule.congr
-    (BilinForm.toLin ≪≫ₗ TensorProduct.lift.equiv A _ _ _)
-    (BilinForm.toLin ≪≫ₗ TensorProduct.lift.equiv R _ _ _)).toLinearMap
+  LinearMap.toBilin.toLinearMap ∘ₗ (LinearMap.tensorDistrib R A ∘ₗ
+    (TensorProduct.AlgebraTensorModule.congr toLin toLin ).toLinearMap)
```

Comment:
You should be able to do something similar to `tensorDistribEquiv` below too.

Reply:
Yes, I was just working on that. Done.